bfabric/bfabric2.py

```diff
@@ -144,7 +181,7 @@ def auth(self) -> BfabricAuth:
         return self._auth
```

Comment:
Shouldn't it return self._config?

Reply:
For a second I was questioning everything...

<img width="504" alt="grafik" src="https://github.com/fgcz/bfabricPy/assets/9108636/e45a29c2-5628-4921-919d-1eb75aaf6c8b">

GitHub is very misleading however, you can see the current method name is `def auth(self)` and it collapsed some contents of the file.

Reply:
Damn, my bad. I need to research a better way to do review, preferably without collapsing at all.